lib/Graph/Nodes.cpp

```diff
@@ -200,7 +200,14 @@ static bool verifyPool(NodeValue src, NodeValue dest,
   ShapeNHWC exp(idim.n, outSz.first, outSz.second, idim.c);
   isValid &=
       expectCompareTrue("Unexpected output dimensions", exp, odim, parent);
-  isValid &= checkTypeIgnoreShape(src, dest, dest.getNode());
+  if (src.getType()->isQuantizedType() && isAvgPool) {
+    // For quantized AvgPool, the scale and offset of its input and output could
```

Comment:
Is this documented anywhere? Like, in the doc string of the node?

Reply:
No, it hasn't been documented yet. However, after this PR is merged, we need to update our quantization docs.